### ISSUE TITLE
Inheriting cookiecutters

### ISSUE THREAD

**Issue by apra93** [audreyr/cookiecutter#1118]
I hope I didn't miss this somewhere in the docs but is there an easy way to effectively *subclass* cookiecutters? 

For example, I have cookiecutter A with some folder structure, and now I want to make two new cookiecutters, B and C which extend A and are distinct from each other. I'm imagining this would be implemented using submodules or some way which propagates changes in A to B and C.

**Comment by pydanny** [comment]
Template inheritance in this way doesn't exist in Cookiecutter.

**Comment by pydanny** [comment]
No, there's no way to make it work. It's out of scope for the project.
